model/api-model.js

```diff
@@ -0,0 +1,9 @@
+const fs = require("fs/promises");
+
+exports.readEndpoints = () => {
+  return fs.readFile("endpoints.json", "utf8")
+  .then((response) => {
+    const parsedResponse = JSON.parse(response);
+    return parsedResponse;
+  });

```

Comment:
Looks good!

If you wanted to you could keep the readFile logic in the controller instead.

As we're not interacting with the DB we don't really need a model :)